ruby-challenges.rb

```diff
@@ -22,7 +39,21 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+# Pseudocode: 1.) Create a method called de_voweler
+# 2.) de_voweler will take in a string argument
+# 3.) use .delete method to target voewls to be removed
+# 4.) use ! bang operator to mutate the string
+# 5.) return the mutated string
+
+def de_voweler string
+    string.delete! "aeiouy"
+end
 
```

Comment:
Great job using the `.delete` method. However, what happens when you plug in `beatles_album3` into your method? Remember to handle edge cases, such as the capital vowel that is in `beatles_album3`!